spec/docker_test/network_spec.rb

```diff
@@ -59,4 +59,61 @@
       )
     end
   end
+
+  context 'testing to connect container to network' do
+    it 'created a container' do
+      expect(chef_run).to run_docker_container('network-container').with(
+        repo: 'alpine',
+        tag: '4.1',
+        command: 'sleep 120'
+      )
+    end
+
+    it 'created the network we are connecting to a container' do
+      expect(chef_run).to create_docker_network('test-network-connect').with(
+        container: 'network-container'
+      )
+    end
+
+    it 'connects container to network' do
+      expect(chef_run).to connect_docker_network('test-network-connect').with(
+        container: 'network-container'
+      )
+    end
+  end
+
+  context 'testing ip range' do
+    it 'should set a ip range' do
+      expect(chef_run).to create_docker_netwrok('test-network-ip-range').with(
```

Comment:
Yep. On it...
I'm going to spend the day getting the network and volume bits polished.

On Tue, Feb 16, 2016 at 11:33 AM, Brent Walker <notifications@github.com>
wrote:

> In spec/docker_test/network_spec.rb
> <https://github.com/chef-cookbooks/docker/pull/646#discussion_r53036424>:
>
> > +    it 'created the network we are connecting to a container' do
> > +      expect(chef_run).to create_docker_network('test-network-connect').with(
> > +        container: 'network-container'
> > +      )
> > +    end
> > +
> > +    it 'connects container to network' do
> > +      expect(chef_run).to connect_docker_network('test-network-connect').with(
> > +        container: 'network-container'
> > +      )
> > +    end
> > +  end
> > +
> > +  context 'testing ip range' do
> > +    it 'should set a ip range' do
> > +      expect(chef_run).to create_docker_netwrok('test-network-ip-range').with(
>
> Typo here
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/chef-cookbooks/docker/pull/646/files#r53036424>.
>